implementation/objectoriented.tex

```diff
@@ -47,8 +50,8 @@ \subsection{Memory Mapped I/O}
   \label{fig:back:memorymapped}
 \end{figure}
 
-\todo{Is the table supposed to look like this? it seems broken}
-A subsection of the \gls{adc} control register is given in \autoref{fig:back:memorymapped}
+\autoref{fig:back:memorymapped} shows a subsection of RAM containing the control register \gls{adc}.
```

Comment:
kanskje  bytte om på plassering av ADC: `shows a subsection of RAM that contains the ADC control register`.